src/base/static/state/ducks/map.js

```diff
@@ -43,6 +46,7 @@ const INIT_LAYER = "map/INIT_LAYER";
 const UPDATE_LAYERS_VISIBLE = "map/UPDATE_LAYERS_VISIBLE";
 const UPDATE_LAYERS_HIDDEN = "map/UPDATE_LAYERS_HIDDEN";
 const UPDATE_MAP_DRAGGING = "map/UPDATE_MAP_DRAGGING";
+const SET_IS_MAP_DRAGGED = "map/SET_IS_MAP_DRAGGED";
```

Comment:
Let's rename `SET` to `UPDATE`.

details here: https://github.com/erikras/ducks-modular-redux

`setIsMapDragged` -> `updateIsMapDragged`

Just like how Create and Update are separate CRUD actions, this will clarify that we aren't setting/initializing it for the first time, but updating an existing value.